DOC Deprecate CMS GraphQL endpoints (#559)
silverstripe/developer-docs: en/08_Changelogs/5.3.0.md

- The [`RememberLoginHash::renew()`](api:SilverStripe\Security\RememberLoginHash::renew()) method has been deprecated without replacement, since the associated behaviour will be removed in 6.0.
  - The `onAfterRenewToken` extension point within this method will likely be replaced with a new extension point in 6.0.
- The [`RememberLoginHash.replace_token_during_session_renewal`](api:SilverStripe\Security\RememberLoginHash->replace_token_during_session_renewal) configuration property has been added to allow disabling token regeneration during session renewal. This property will be removed in 6.0.

> - Code for the CMS GraphQL `admin` schema which provided endpoints for the CMS has been deprecated and will be removed in CMS 6. Functionality the GraphQL endpoints was responsible for has been replaced with regular Silverstripe controller endpoints instead. Extension hooks have been added to the new controller endpoints that return data to allow for customisation. Frontend schemas, such the `default` schema, will continue to work in CMS 6.

Reviewer: ```suggestion
- Code for the CMS GraphQL `admin` schema which provided endpoints for the CMS has been deprecated and will be removed in CMS 6. Functionality which the GraphQL endpoints are currently responsible for will be replaced with regular Silverstripe controller endpoints instead. Extension hooks will be added to the new controller endpoints that return data to allow for customisation. Frontend schemas, such the `default` schema, will continue to work in CMS 6.
```
Future tense, so it's clear this hasn't happened in CMS 5.3 - this is just meant to be a heads up about the deprecation.

Author: Updated
